torch/_custom_op.py

```diff
@@ -370,3 +449,66 @@ def validate_function_matches_schema(
             f"Schema has kwonlyarg names {kwonlyarg_names} but function has "
             f"{arg_spec.kwonlyargs}."
         )
+
+
+# Global dictionary holding weak references to all CustomOp objects
+# Used to query the CustomOp associated with a specific C++ dispatcher operator.
+# An example usage is FakeTensor: FakeTensor checks if a specific operator
+# has an implementation registered via the CustomOp API.
+global_registry: weakref.WeakValueDictionary = weakref.WeakValueDictionary({})
+
+
+def get_none():
+    return None
+
+
+global_ctx_getter: typing.Callable = get_none
```

Comment:
Is there are particular reason this is a callable, as opposed to just holding the context itself?